### ISSUE TITLE
Incorrect field calibration JSON prediction file present under game folder

### ISSUE THREAD

**Issue by PranshuAgrawal86** [SilvioGiancola/SoccerNetv2-DevKit#16]
Hi,

Wrong field calibration file seems to be present under a different game folder. Hence, no. of frames extracted for a 25fps video will not match with no of predictions present inside field_calibration.json because 'UrlLocal' points to a different game video. 

Example - 
Refer to Game : 
england_epl/2014-2015/2015-05-17 - 18-00 Manchester United 1 - 1 Arsenal
[https://exrcsdrive.kaust.edu.sa/exrcsdrive/index.php/s/9eRjic29XTk0gS9?path=%2Fengland_epl%2F2014-2015](url)

Check file-  
1_field_calibration.json , it contains "UrlLocal": "/ibex/scratch/giancos/SoccerNet_field/england_epl/2014-2015/2015-04-11 - 19-30 Burnley 0 - 1 Arsenal/1_HQ_25.mkv",

2_field_calibration.json, it contains "UrlLocal": "/ibex/scratch/giancos/SoccerNet_field/england_epl/2015-2016/2015-08-08 - 19-30 Chelsea 2 - 2 Swansea/2_HQ_25.mkv",

There are 217 such instances out of 1000 where such incorrect mapping of game folder to game video is present.

**Comment by cioppaanthony** [comment]
Hi @PranshuAgrawal86,

Sorry for the late reply, I was on vacation. 
That is a really weird error. I have checked locally on my workstation and don't have the issue. The game that you mention is correctly linked to the correct calibration file. 

But then, I tried to download it directly from the pip package using:
```
from SoccerNet.Downloader import SoccerNetDownloader
mySoccerNetDownloader = SoccerNetDownloader(LocalDirectory="/path/to/soccernet")
mySoccerNetDownloader.downloadGames(files=["1_field_calibration_ccbv.json", "2_field_calibration_ccbv.json"], split=["train","valid","test"]) 
```
and got the same error as you did... 
I will investigate that with @SilvioGiancola who manages the SoccerNet pip package. There has probably been an error when uploading the files to our server. Sorry about that.

**Comment by SilvioGiancola** [comment]
Hi @PranshuAgrawal86 ,

Thank you for raising such issue, and sorry for the delay in the answer, it was holiday time :) 

I fixed this issue in our latest [commit](https://github.com/SilvioGiancola/SoccerNetv2-DevKit/commit/46560c7b2c6fa65d9cc67514bf889694fd9641a8). For some reason, the field calibration features were broken/missing in our server. I couldn't update them, they were somehow "locked". I had to upload a new set of cleaned features, they now go under the name `*_field_calib_ccbv.json`. Make sure you also use `*_player_boundingbox_maskrcnn.json` for the player bounding boxes. I tried the code on a fresh install and I was able to download the field calibration and train a model.

Let us know if you still have an issue, I can follow up with you if needed.

Cheers,
